scripts/populate_institutions.py

```diff
@@ -243,6 +243,18 @@ def main(default_args=False):
                 'email_domains': [],
                 'delegation_protocol': 'saml-shib',
             },
+            {
+                '_id': 'cord',
+                'name': 'Concordia College',
+                'description': '<a href="https://www.concordiacollege.edu/">Concordia College</a> | <a href="https://www.concordiacollege.edu/academics/library/">Carl B. Ylvisaker Library</a> | <a href="https://cord.libguides.com/?b=s">Research Guides</a>',
+                'banner_name': 'cord-banner.png',
+                'logo_name': 'cord-shield.png',
+                'login_url': None,
```

Comment:
As mentioned in https://github.com/CenterForOpenScience/cas-overlay/pull/178, institution SSO using the CAS / pac4j does not have a pre-configured login URL stored in the DB.